.github/workflows/lint-and-test.yml

```
@@ -0,0 +1,45 @@
+## Reference: https://github.com/helm/chart-testing-action
+---
+name: Linting and Testing
+on: pull_request
+jobs: 
+  chart-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+
+      - uses: actions/setup-python@v2
```

Maybe add a friendly name inside the pipeline for this?
```suggestion
      - name: Setup python
        uses: actions/setup-python@v2
```